geomesa-spark/geomesa-spark-sql/src/test/scala/org/locationtech/geomesa/spark/SparkSQLTestUtils.scala

```diff
@@ -88,5 +88,26 @@ object SparkSQLTestUtils {
     val fs = ds.getFeatureSource(name).asInstanceOf[SimpleFeatureStore]
     fs.addFeatures(features)
   }
+
+  def generatePoints(gf: GeometryFactory, numPoints: Int): Map[String, String] = {
+    (1 until numPoints).map { i =>
+      val x = -180 + 360 * Math.random()
```

Comment:
for all these random numbers, it would be good to create a Random with a defined seed up front so that the test uses the same values each time it's run.